.github/workflows/ci.yaml

```diff
@@ -86,3 +86,5 @@ jobs:
       HOBBYFARM_UI_HEADER_TITLE: Rancher HobbyFarm
       HOBBYFARM_UI_URL: https://learn.hf.local
       HOBBYFARM_GARGANTUA_URL: https://api.hf.local
+      HOBBYFARM_ADMIN_UI_PWD_NEW: admin
+      HOBBYFARM_ADMIN_UI_HASHPWD_NEW: "$2a$10$33fQs0G.lHQdDAsdoECgA.8iYvNtyJ2XC2AmvR5x6ZkzxSuKXyfFM"
```

Comment:
HOBBYFARM_ADMIN_UI_HASHPWD_NEW is not mandatory
The other hashpassword is only used locally but no issue I can delete this line afterwards, it has no impact